CMakeLists.txt

```diff
@@ -619,6 +619,17 @@ set_target_properties(nearCacheTest PROPERTIES COMPILE_DEFINITIONS "${DLLEXPORT_
 set_target_properties (nearCacheTest PROPERTIES COMPILE_FLAGS "${COMPILER_FLAGS} ${WARNING_FLAGS_NO_PEDANTIC} ${NO_UNUSED_FLAGS}")
 target_link_libraries (nearCacheTest hotrod hotrod_protobuf ${PROTOBUF_LIBRARY} ${platform_libs})
 
+add_executable (nearCacheFailoverTest test/NearCacheFailoverTest.cpp)
```

Comment:
@rigazilla, I think the `test/NearCacheFailoverTest.cpp` file is missing from this commit

Reply:
@alanfx yep, added. just for linux atm, the windows version needs some tuning

Reply:
@alanfx should be ok now, please have a look